dev/com.ibm.ws.repository.generator/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+task copyEsas(type: Copy) {
+    dependsOn parent.subprojects.assemble
+    from cnf.fileTree(".").include("/release/dev/*/*/*.esa").files
+    into 'build/esas'
+}
+
+task createSingleJson {
+    // Run after all feature ESAs have been built
+    dependsOn copyEsas
+    doLast {
+        ant.taskdef(name:'singleJsonRepo', classname:'generator.SingleJsonRepositoryGenerator') {
```

Comment:
Does this work?  Looks like classname should be com.ibm.ws.repository.generator.SingleJsonRepositoryGenerator